index.xml

```diff
@@ -14044,66 +14181,126 @@ C implements A;</x:codeblock>
-                      and set <var>count</var> to <var>i</var> + 1.</li>
-                    <li>Set <var>i</var> to <var>i</var> + 1.</li>
-                  </ol>
-                </li>
-                <li>Truncate <var>values</var> to have length <var>count</var>.</li>
-                <li>Let <var>script</var> be the <a class='dfnref' href='#dfn-callback-context'>callback context</a> associated with <var>V</var>.</li>
-                <li>Push <var>script</var> on to the <a class='dfnref external' href='http://www.whatwg.org/specs/web-apps/current-work/multipage/webappapis.html'>stack of incumbent scripts</a>. <a href='#ref-HTML'>[HTML]</a></li>
-                <li>Try running the following step:
-                  <ol>
-                    <li>Set <var>R</var> to the result of invoking the <span class='prop'>[[Call]]</span> method of <var>F</var>, providing the <a class='dfnref' href='#dfn-callback-this-value'>callback this value</a> as the <span class='esvalue'>this</span> value and <var>values</var> as the argument values.</li>
-                  </ol>
-                  And then, whether or not an exception was thrown:
-                  <ol>
-                    <li>Pop <var>script</var> off the <a class='dfnref external' href='http://www.whatwg.org/specs/web-apps/current-work/multipage/webappapis.html'>stack of incumbent scripts</a>.</li>
-                    <li>If an exception was thrown, end these steps, and allow it to propagate.</li>
-                  </ol>
-                </li>
-                </ol>
-                </li>
-                <li>If the callback function’s return type is <span class='idltype'>void</span>, return.</li>
-                <li>
-                  Return the result of <a class='dfnref' href='#dfn-convert-ecmascript-to-idl-value'>converting</a>
-                  <var>R</var> to an IDL value of the same type as the callback function’s return type.
-                </li>
+                <li>If the callback function's return type is <span class='idltype'>void</span>,
+                return.</li>
+
+                <li>Return the result of <a class='dfnref'
+                href='#dfn-convert-ecmascript-to-idl-value'>converting</a> <span
+                class='esvalue'>undefined</span> to the callback function's return type.</li>
               </ol>
-              And then, if an exception was thrown:
+            </li>
+
+            <li>Let <var>realm</var> be <var>F</var>'s <a class='dfnref'
+            href='#dfn-associated-realm'>associated Realm</a>.</li>
+
+            <li>Let <var>settings</var> be <var>realm</var>'s <a class='dfnref external'
+            href='https://html.spec.whatwg.org/multipage/webappapis.html#concept-realm-settings-object'>settings
+            object</a>.</li>
+
+            <li><a>Prepare to run script</a> with <var>settings</var>.</li>
+
+            <li>If <var>value</var>'s interface is not a <a class='dfnref'
```

Comment:
This looks like copy/paste fail.  there is no "`value`'s interface" and no `O` here.  This entire step is not needed; we already set `thisArg` to undefined if not provided, which is the behavior we want for the callback function case.